src/lib/swapper/swappers/ThorchainSwapper/buildThorTrade/buildThorTrade.ts

```diff
@@ -133,6 +138,10 @@ export const buildTrade = async (
       txData: buildTxResponse.txToSign,
     })
   } else if (chainNamespace === CHAIN_NAMESPACE.CosmosSdk) {
+    // TODO(gomes): it is not - plumb from through
```

Comment:
Actual TODO, to be akschually tackled in a follow-up, reduces the risk area like so